server/config/postgresql.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  user: 'elvinkosova',
+  host: 'localhost',
+  database: 'elvinkosova',
+  //password: '',
+  port: 5432,
+};
```

Comment:
I wonder if you want to hide this information (gitignore) ?

Reply:
I decided to use it as a template for production. I don't expect the structure of it to change in the future.